CONTRIBUTING.md

```diff
@@ -27,12 +27,12 @@ This bundle is tested with [atoum](https://github.com/atoum/atoum).
 
 To launch tests, you need to install dependencies in dev mode
 ```bash
-composer install --dev
+composer install
 ```
 
 And you can now launch tests
 ```bash
-./vendor/bin/atoum
+./bin/atoum
```

Comment:
you could also remove the `./` to only keep `bin/atoum`

Reply:
Done.